src/crypto/utils/helpers.py

```diff
@@ -1,4 +1,13 @@
+import json
+
 from google.oauth2 import service_account
+from google.oauth2.service_account import Credentials
+
+
+def service_credentials(service_acct: dict) -> Credentials:
+    return service_account.Credentials.from_service_account_info(info=service_acct)
 
-def service_credentials(service_acct: dict):
-    return service_account.Credentials.from_service_account_file(service_acct)
\ No newline at end of file
+def parse_service_account_file(path: str) -> dict:
```

Comment:
I guess subtyping would be a hell right? so leaving it generic?

that's nice about typing, you can add as little or much as you want ...

Reply:
yeah, that's correct. There have been a few spots where thought could be more specific on the typing but don't really want to go that deep.